### PR TITLE
BrickHack Club Site Design Fixes

### LINKED ISSUE #1230: BrickHack Club Site - Design Critiques
- [x] make dropdown button be #ffffff white on default
- [x] making hover cover of .btn-dark be dark blue (check my dms with claire for verification)
- [x] put underline on the nonhovered text links
- [x] make button width the same for each team
- [x] body text should have more vertical spacing (24 or 26 for a 20pt font) 
- [x] bigger line height on the nav links
- [x] decrease headings to 64px font size
- [x] fix the heading alignment on the left
- [x] fix the mobile heading overflow
- [x] increase padding on the left and right of the buttons
- [x] move content to the left, make navbar 20% width
- [x] more space under the email us button in contact
- [x] align bottom of button and image together on the events section when side by side
- [x] footer logo a bit smaller, bottom edge line up with the bottom edge of the address 
- [x] teams section move image down more? and move text to the left, decrease that gap
- [x] meta tags 
- [x] add a twitter and maybe a reddit and maybe 
- [x] fix content jumping on mobile menu open/close

### PR DESCRIPTION
**Pull request by sjmiller7**
Fixes #1230 

~~One thing left to do (navbar jumping issue)~~ Done!

### CODE REVIEW

**Comment by cbaudouinjr** [comment]
its got some divs all looks good 👍

**Review by cbaudouinjr — DISMISSED** [review]
its got some divs all looks good 👍

**Review by peterkos — APPROVED** [review]
LGTM! Nit in the design channel but its design related

**Comment by clairegallup** [comment]
I probably should have noticed this earlier, but the mobile navbar feels too tall IMO, and its contents aren't vertically centered

image

I think it'd look better if the amount of space currently on the bottom of the navbar (or maybe even a little less) was used for the top and bottom, since that'd solve both problems. Thoughts?

File: sass/club.scss [hunk 1]
```diff
@@ -132,12 +132,11 @@ nav {
 
     .nav-link {
         font-size: 1.25rem;
-        line-height: 2.5rem;
-        margin: 10px 0 14px;
+        margin: 10px 0 19px;
```
Comment (peterkos) [inline sass/club.scss:135]: 19px?

File: sass/club.scss [hunk 2]
```diff
@@ -280,7 +279,8 @@ section {
 .ui-selectmenu-button {
     appearance: none;
     outline: none;
-    width: 18rem; // best approximation of the largest team name
+    width: max-content;
+    max-width: calc(100vw - 3.5rem); // Not using 100%
```
Comment (peterkos) [inline sass/club.scss:283]: can 3.5rem be a var (page margin?)

File: sass/club.scss [hunk 3]
```diff
@@ -396,7 +397,7 @@ section {
 
     img {
         max-width: 100%;
-        margin-top: 5rem;
+        //margin-top: 5rem;
```
Comment (peterkos) [inline sass/club.scss:400]: nit

**Review by peterkos — COMMENTED** [review]
LGTM but a couple nits

**Review by peterkos — APPROVED** [review]

**Pull request merged**